client/src/api/notifications.js

```diff
@@ -0,0 +1,41 @@
+import httpClient from './httpClient';
+
+export const getNotifications = async (userId) => {
```

Comment:
is the injection still not working for you? It works for me. Probably something we can sort out to cut down on duplicate code.

Reply:
Okay Ive removed it for now and we'll see how it goes. No issue at the moment